core/crypto/bls/bls.go

```diff
@@ -51,34 +47,28 @@ func AggregateSignatures(signaturesWithPublicKey ...SignatureWithPublicKey) (agg
 
 	mask, err := sign.NewMask(blsSuite, publicKeyPoints, nil)
 	if err != nil {
-		err = xerrors.Errorf("failed to create mask (%v): %w", err, ErrBLSFailed)
-
-		return
+		return SignatureWithPublicKey{}, xerrors.Errorf("failed to create mask (%v): %w", err, ErrBLSFailed)
```

Comment:
here `xerrors` is used. Shouldn't we make this consistent?

Reply:
IMO we should make this consistent, but maybe not in the same PR?

Reply:
Yeah probably better to do it in a separate PR sometime